### DISCUSSION TITLE
Github Pages Reload Issue

### DISCUSSION THREAD

**Discussion by jeremybrachle** [community/community#22901] [Other Feature Feedback, Questions, & Ideas]
<p>I have a github pages site set up at <a href="https://www.jeremybrachle.com/" rel="nofollow noopener">https://www.jeremybrachle.com/</a> and it works really well for the most part. The only problem I have is when I go to a page other than home (for example: <a href="https://www.jeremybrachle.com/about" rel="nofollow noopener">https://www.jeremybrachle.com/about</a>) and click reload, I get an error on the console saying “Refused to load the image ‘<a href="https://www.jeremybrachle.com/favicon.ico" rel="nofollow noopener">https://www.jeremybrachle.com/favicon.ico</a>’ because it violates the following Content Security Policy directive: “img-src data:”.” and the browser displays a 404 error. I was able to get it to work by using the hash location strategy, but I don’t want hashes displayed in my URL when navigating to different pages. Is this something I could get to work with a Jekyll plug in?</p>

**Comment by angela-crist** [comment]
<p>Hi <a class="mention" href="/jeremybrachle">@jeremybrachle</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now here in the GitHub Pages board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Cheers!</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Taking a look at your source at <a href="https://github.com/jeremybrachle/jeremybrachle.github.io" rel="nofollow noopener">https://github.com/jeremybrachle/jeremybrachle.github.io</a>, it appears that you’ve built an Angular site? What you’ve done is you’ve created routing handlers for the various header links (<code>about</code>, <code>experience</code>, and <code>contact</code>) but no actual pages for those. So when you click the “About” link in the header on the home page, the Angular code does some JavaScript magic to render the about page and make it <em>look</em> like you’ve navigated to another page … but the page doesn’t exist except in the memory of the browser. So when you click reload, the browser faithfully attempts to load <a href="https://www.jeremybrachle.com/about" rel="nofollow noopener">https://www.jeremybrachle.com/about</a> and the webserver accurately responds with a 404, because that page doesn’t, in fact, exist.</p>
<p>What you would have to do is perhaps create an about.html that loads your Angular app and renders the “about page” template you have. The same would have to be done for the other pages.</p>
<p>I hope that helps!</p>

**Reply by husainazkas** [reply]
I have same problem with Flutter Web.

According to your answer, I can confirm that it's true. But, I don't think it's a best practice to solve this problem. Hope anyone else can provide another good solution here. Of course, thanks for trying to give solution.
